Add SBMTD Mobility Pass vars to Terraform

machikoyasuda opened the pull request:
closes #1777

thekaveman reviewed (commented):
Can you please confirm these are all added and configured in the 3 different environments (`dev`, `test`, `prod` Key Vaults in Azure?)

machikoyasuda commented:
Secrets status:
- Dev: Added all new variables for SBMTD, including FrontDoor URL for SBMTD Eligibility Server (Dev), added `mst-server-public-key-url`. Set `mobility-pass-verifier-active` to `True`. Removed `sever-public-key-url`.
- Test and Prod: Added all new variables for SBMTD, added `mst-server-public-key-url`, does not have FrontDoor URLs for Test/Prod envs for Eligibility Server. Set `mobility-pass-verifier-active` to `False`. Did NOT remove `sever-public-key-url` b/c it is still being used.

machikoyasuda commented:
@thekaveman @angela-tran Need to get this PR re-reviewed, before I can continue set up of SBMTD Mob Pass on Dev.

angela-tran commented on terraform/app_service.tf:
```diff
@@ -157,6 +157,14 @@ resource "azurerm_linux_web_app" "main" {
+    "MOBILITY_PASS_VERIFIER_ACTIVE"                        = "${local.secret_prefix}mobility-pass-verifier-active)"
+    "MOBILITY_PASS_VERIFIER_API_URL"                       = "${local.secret_prefix}mobility-pass-verifier-api-url)"
+    "MOBILITY_PASS_VERIFIER_API_AUTH_HEADER"               = "${local.secret_prefix}mobility-pass-verifier-api-auth-header)"
+    "MOBILITY_PASS_VERIFIER_API_AUTH_KEY"                  = "${local.secret_prefix}mobility-pass-verifier-api-auth-header)"
```
```suggestion
    "MOBILITY_PASS_VERIFIER_API_AUTH_KEY"                  = "${local.secret_prefix}mobility-pass-verifier-api-auth-key)"
```
